src/pages/Main/Components/LectureSlide.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import Card from '../../../components/Card/Card';
+
+const LectureSlide = () => {
+  const [cardList, setCardList] = useState([]);
+  const getData = async () => {
+    const data = await (await fetch('/data/cardList.json')).json();
+
+    setCardList(data);
+  };
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const settings = {
+    slide: <Card />,
+    dots: true,
+    infinite: true,
+    speed: 500,
+    arrows: true,
+    autoplay: false,
+    autoplaySpeed: 2000,
+    slidesToShow: 3,
+    slidesToScroll: 1,
+    centerMode: true,
+    variableWidth: true,
+    centerPadding: '0px',
+  };
+
+  return (
+    <Wrapper>
+      <StyledSlide {...settings}>
+        {cardList && cardList.map(card => <Card key={card.id} card={card} />)}
```

Comment:
```suggestion
        {cardList?.map(card => <Card key={card.id} card={card} />)}
```
&&를 이용한 단축평가와 동일한 동작을 optional chaining 문법 이용해서 좀 더 간결하게 표현할 수도 있습니다. [공식문서](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining) 참고해서 어떻게 동작하는지 확인 및 이해해보시고 적용해보세요! 

Reply:
우와!! 감사합니다!